content/en/developers/dogstatsd/dogstatsd_mapper.md

```diff
@@ -0,0 +1,129 @@
+1. [Open your `datadog.yaml` file][1].
+2. Add a [mapping rule configuration block](#mapping-rule-configuration) under the `dogstatsd_mapper_profiles` parameter.
+
+## Mapping rule configuration
+
+A mapping rule block has the following layout:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: '<PROFILE_NAME>'
+      prefix: '<PROFILE_PREFIX>'
+      mappings:
+          - match: '<METRIC_TO_MATCH>'
+            match_type: '<MATCH_TYPE>'
+            name: '<MAPPED_METRIC_NAME>'
+            tags:
+                '<TAG_KEY>': '<TAG_VALUE_TO_EXPAND>'
+```
+
+With the following placeholders:
+
+| Placeholder             |  Definition                                                                                                                               | Required                |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+|  `<PROFILE_NAME>`       | A name to give to your mapping rule profile.                                                                                              | yes                     |
+| `<PROFILE_PREFIX>`      | The metric name prefix associated to this profile.                                                                                        | yes                     |
+| `<METRIC_TO_MATCH>`     | The metric name to extract groups from with the [Wildcard](#wildcard-match-pattern) or [Regex](#regex-match-pattern) match logic.         | yes                     |
+| `<MATCH_TYPE>`          | The type of match to apply to the `<METRIC_TO_MATCH>`. Either [`wildcard`](#wildcard-match-pattern) or [`regex`](#regex-match-pattern)    | no, default: `wildcard` |
+| `<MAPPED_METRIC_NAME>`  | The new metric name to send to Datadog with the tags defined in the same group.                                                           | yes                     |
+| `<TAG_KEY>`             | The Tag key to associate to the tags collected.                                                                                           | no                      |
+| `<TAG_VALUE_TO_EXPAND>` | The tags collected from the `<MATCH_TYPE>` to inline.                                                                                     | no                      |
+
+## Wildcard Match Pattern
+
+The wildcard match pattern matches dot-separated metric names using `*` as wildcard. The metric name must be only composed of alphanumeric, `.`, and `_` characters for this pattern to work.
+Groups extracted can then be expanded with the `$n` format e.g. `$1`, `$2`, `$3`... or the `${n}` format e.g. `${1}`, `${2}`, `${3}`, ...
+
+For instance, if you have the metric `custom_metric.value_1.value_2` with the following mapping group configuration:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: my_custom_metric_profile
+      prefix: custom_metric
+      mappings:
+          - match: 'custom_metric.*'
```

Comment:
Since 2 wildcards needed since you use $1 and $2. I suggest adding `process` or any metric prefix, more realistic.
```suggestion
          - match: 'custom_metric.process.*..*'
```